### PR TITLE
Update docs

### PR DESCRIPTION
**Pull request by jcfr**

### CODE REVIEW

**Comment by jcfr** [comment]
Cc: @thewtex

File: docs/usage.rst [hunk 1]
```diff
@@ -1,3 +1,15 @@
+
+===============================
+Why should I use scikit-build ?
```
Comment (thewtex) [inline docs/usage.rst]: Cool 8-)

File: docs/usage.rst [hunk 2]
```diff
@@ -76,6 +115,38 @@ build tool options
...
+
+See `CMake Toolchains <https://cmake.org/cmake/help/v3.6/manual/cmake-toolchains.7.html>`_.
+
+
+Introduction to dockross
+^^^^^^^^^^^^^^^^^^^^^^^^
+
+.. note:: *To be documented.*
+
+    See https://github.com/scikit-build/scikit-build/issues/80
```
Comment (thewtex) [inline docs/usage.rst]: Will take a stab.

**Comment by jcfr** [comment]
Thanks for the review

**Pull request merged**